uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -279,36 +279,33 @@ <h4 class="modal-title">
                   </span>
                 </ul>
               </div>
-            <div class="chr_leave-request-files_upload" ng-disabled="$ctrl.isMode('view')">
-              <div
-                nv-file-drop
-                uploader="$ctrl.request.fileUploader"
-                ng-disabled="$ctrl.isMode('view')"
-                ng-if="$ctrl.request.fileUploader && $ctrl.canUploadMore()">
-                <div class="chr_leave-request-files_upload_action row text-center">
-                  Drag and drop file here
-                </div>
-                <div class="chr_leave-request-files_upload_action row text-center">
-                  or
+              <div class="chr_leave-request-files_upload" ng-disabled="$ctrl.isMode('view')">
+                <div
+                  nv-file-drop
+                  uploader="$ctrl.request.fileUploader"
+                  ng-if="$ctrl.request.fileUploader && ($ctrl.canUploadMore() || $ctrl.userRemovedFile)">
```

Comment:
Are we removing the uploader if the user has already uploaded 10 files? I think we should not, because as @jamienovick mentioned, options should not be hidden, because it might confuse the user, we can just disabled it and show an related error message. This deserves a discussion on Leave and Absence room.